src/worker/tasks/sendTransactionWorker.ts

```diff
@@ -393,3 +381,43 @@ export const initSendTransactionWorker = () => {
     }
   });
 };
+
+export const getPopulatedOrErroredTransaction = async (
+  queuedTransaction: QueuedTransaction,
+): Promise<
+  | [Awaited<ReturnType<typeof toSerializableTransaction>>, undefined]
+  | [undefined, ErroredTransaction]
+> => {
+  try {
+    const from = queuedTransaction.isUserOp
+      ? queuedTransaction.accountAddress
+      : queuedTransaction.from;
+
+    if (!from) throw new Error("Invalid transaction parameters: from");
+
+    const populatedTransaction = await toSerializableTransaction({
+      from: getChecksumAddress(
+        queuedTransaction.isUserOp
+          ? queuedTransaction.accountAddress
+          : queuedTransaction.from,
```

Comment:
could reuse the `from` variable here no?

Reply:
nice catch! thanks